group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
```

Comment:
Ah I see. Man, I still have some things to sort out in my mind on this. Maybe this confusion is my personal problem, not a general problem. If so, it may annul my other comment re: `my-bedrock`.